### PR TITLE
update bot account creation and UI sections

### PR DESCRIPTION
**Pull request by ewwollesen**
#### Summary
I updated the Bot Account Creation and  User Interface section to add instructions on adding the bot account to a Team, and also took the liberty of adding information on saving the bot account key as I felt that was missing as well.

I wasn't really sure the best place to put the team add info, so I just added it on to an existing sentence rather than create its own section because it seemed simple enough that it didn't need its own space. If you want it to have its own section though I could certainly do that.

#### Ticket Link

https://github.com/mattermost/docs/issues/3859

### CODE REVIEW

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Awesome change, thank you! I've added a PM for an additional review. :)

File: site/content/integrate/admin-guide/admin-bot-accounts/_index.md [hunk 1]
```diff
@@ -41,17 +41,19 @@ Once set, System Admin can create bot accounts for integrations using the **Inte
 Below are different ways to create bot accounts. After the bot account is created, make sure to:
 
 1. Copy the generated bot access token for your integration.
-2. Add the bot account to teams and channels you want it to interact in.
+2. Add the bot account to teams and channels you want it to interact in. To add the bot account to your team, navigate to **Main Menu > Invite People** and enter the bot account in the **Add or Invite People** field, then choose **Invite Member**. After that you should be able to add the bot account to channels like any other user.
```
Comment (cwarnermm) [inline site/content/integrate/admin-guide/admin-bot-accounts/_index.md:44]:
```suggestion
2. Add the bot account to teams and channels you want it to interact in by going to **Main Menu > Invite People**, entering the bot account in the **Add or Invite People** field, then choosing **Invite Member**. You should now be able to add the bot account to channels like any other user.
```

**Review by cwarnermm — COMMENTED** [review]
Thank you so much for this docs contribution. I've suggested one non-blocking update to consider.

**Comment by ewwollesen** [comment]
> Thank you so much for this docs contribution. I've suggested one non-blocking update to consider.

That looks better. I went ahead and merged that in.

**Review by justinegeffen — APPROVED** [review]
Awesome work, thank you!!

**Review by cwarnermm — APPROVED** [review]
This is a great update! Thank you so much!

**Pull request merged**